packages/form-js-viewer/src/render/components/form-fields/Button.js

```diff
@@ -1,17 +1,25 @@
 import { formFieldClasses } from '../Util';
-import { useSingleLineTemplateEvaluation } from '../../hooks';
+import { useSingleLineTemplateEvaluation, useService } from '../../hooks';
 
 const type = 'button';
 
 export function Button(props) {
   const { disabled, onFocus, onBlur, field } = props;
-
   const { action = 'submit' } = field;
-
   const evaluatedLabel = useSingleLineTemplateEvaluation(field.label || '', { debug: true });
 
+  const form = useService('form');
+  const { schema } = form._getState();
+
+  const direction = schema?.direction || 'ltr'; // Fetch the direction value from the form schema
+
   return (
-    <div class={formFieldClasses(type)}>
+    <div
+      class={formFieldClasses(type)}
+      style={{
+        direction: direction,
+        fontFamily: 'Vazirmatn, sans-serif',
```

Comment:
We have a css variable that defines fontFamily, so you can set it across the entire application rather easily (in general you'll find we don't use inline styles much, except for alignment). So **if** we're implementing it within the form-js, we should do it in one place only (somewhere in the form root).

For now, because I still need to understand a little bit more about the font stuff, please just remove it. We'll get back to it when everything else is done.

PS: the comments on this file apply across the board.

Reply:
✅

Reply:
I removed the font; but I kindly ask you to reconsider this decision. The font makes no difference to English and the languages that have a similar alphabet to English. But in Persian, it is one of the main (if not the main) reasons why people decide not to use a software.

If you take a look at Github, you'll find out that Github is also using [Vazirmatn](https://github.com/rastikerdar/vazirmatn) for its Persian strings. It is highly admired among open-source community.